pkgs/development/compilers/hare/hare/default.nix

```diff
@@ -4,23 +4,23 @@
 , binutils-unwrapped
 , harePackages
 , makeWrapper
-, qbe
+  #, qbe # We use harec's override of qbe until 1.2 is released
 , scdoc
-, substituteAll
 }:
 
 let
   inherit (harePackages) harec;
+  qbe = harec.qbeUnstable;
```

Comment:
```suggestion
  # we use harec's override of qbe until 1.2 is released
  qbe = harec.qbeUnstable;
```

Reply:
Done --- do note that the file was changed from `pkgs/development/compilers/hare/hare/default.nix` to `pkgs/development/compilers/hare/default.nix`, as per the explanation in the updated description of this PR